CHANGELOG.md

```diff
@@ -8,16 +8,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - [#290](https://github.com/nf-core/funcscan/pull/290) Merged pipeline template of nf-core/tools version 2.9, updated references. (by @jfy133)
-- [#285](https://github.com/nf-core/funcscan/pull/285) Use nf-validation for samplesheet checking and added support for `fna.gz` input FASTA files (by @louperelo, @mirpedrol, @jfy133)
-- [#295](https://github.com/nf-core/funcscan/pull/295) Add Prokka to MultiQC output (by @louperelo)
+- [#285](https://github.com/nf-core/funcscan/pull/285) Use nf-validation for samplesheet checking and added support for `fna.gz` input FASTA files. (by @louperelo, @mirpedrol, @jfy133)
+- [#295](https://github.com/nf-core/funcscan/pull/295) Add Prokka to MultiQC output. (by @louperelo)
 
 ### `Fixed`
 
 - [#296](https://github.com/nf-core/funcscan/pull/296) Fixed empty output when saving prodigal annotations. (reported by @louperelo, fix by @jasmezz)
 - [#297](https://github.com/nf-core/funcscan/pull/297) Added check for empty annotation files prior going into screening. (❤️ to @alexhbnr for requesting, added by @jfy133)
+- [#299](https://github.com/nf-core/funcscan/pull/299) Fixed pigz error with symlinks in Pyrodigal, updated other modules: comBGC, FastQC, GECCO, gunzip, untar. (by @jasmezz)
 
 ### `Dependencies`
 
+| Tool  | Previous version | New version |
+| ----- | ---------------- | ----------- |
+| GECCO | 0.9.2            | 0.9.8       |
```

Comment:
Where are the version updates for the other tools? Have you validated the working correctly?

Reply:
Added comBGC. :+1: The other modules (fastqc, gunzip, pyrodigal, untar) are just changes in the nf-core module, not actually version updates. And yes, all modules work correctly.